### PR TITLE
HDDS-7983. Intermittent OutOfMemoryError in TestOzoneRpcClientWithRatis#testUploadWithStreamAndMemoryMappedBuffer.

### PR DESCRIPTION
**Pull request by szetszwo**
## What changes were proposed in this pull request?

- setDataStreamMinPacketSize to 1 MB in the tests

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-7983

## How was this patch tested?

Fixing existing tests.

### CODE REVIEW

File: hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineStream.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ public void setup() throws Exception {
         MiniOzoneCluster.newBuilder(conf)
             .setNumDatanodes(3)
             .setHbInterval(200)
-            .setDataStreamMinPacketSize(1024)
+            .setDataStreamMinPacketSize(1) // 1MB
```
Comment (adoroszlai) [inline hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineStream.java:127]: This test sets size unit:

https://github.com/apache/ozone/blob/dbd2de9bbbdef0441e0b959081e3ef229c0aaa04/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineStream.java#L132

so this part of the patch should be omitted:

```suggestion
            .setDataStreamMinPacketSize(1024)
```

Comment (szetszwo) [reply hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestContainerStateMachineStream.java]: @adoroszlai , Oops, you are right that the unit is set a few lines below.  Reverted the change.

**Review by adoroszlai — APPROVED** [review]
Thanks @szetszwo for updating the patch.

**Comment by szetszwo** [comment]
@adoroszlai , thanks a lot for reviewing this!

**Pull request merged**